packages/animations/pubspec.yaml

```diff
@@ -9,6 +9,7 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
+  vector_math: ^2.0.8
```

Comment:
This is only used in tests, right? In that case, it should be a dev_dependency and moved to the section below.